packages/next-swc/crates/next-core/src/next_config.rs

```diff
@@ -353,6 +353,7 @@ pub struct ExperimentalConfig {
     pub app_dir: Option<bool>,
     pub server_components_external_packages: Option<Vec<String>>,
     pub turbo: Option<ExperimentalTurboConfig>,
+    pub mdx_rs: Option<bool>,
```

Comment:
Nit: expose this as a turbo method instead of public, sot that unrelated changes to the next config don't necessarily invalidate the entire graph.